<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q7MainWindow</class>
 <widget class="QMainWindow" name="Q7MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>218</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>255</horstretch>
    <verstretch>255</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>793</width>
    <height>218</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>793</width>
    <height>218</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../R/Res.qrc">
    <normaloff>:/images/icons/cgSpy.gif</normaloff>:/images/icons/cgSpy.gif</iconset>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="bResetScrollBars">
    <property name="geometry">
     <rect>
      <x>773</x>
      <y>202</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Reset scrollbars to left top corner of window</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../R/Res.qrc">
      <normaloff>:/images/icons/add.gif</normaloff>:/images/icons/add.gif</iconset>
    </property>
   </widget>
   <widget class="QTableWidget" name="Q7ControlTable">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>36</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>781</width>
      <height>181</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>781</width>
      <height>181</height>
     </size>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QToolButton" name="bTreeLoad">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load an existing CGNS file</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../R/Res.qrc">
      <normaloff>:/images/icons/tree-load.gif</normaloff>:/images/icons/tree-load.gif</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="bEditTree">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Create a new CGNS tree from scratch</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../R/Res.qrc">
      <normaloff>:/images/icons/tree-new.gif</normaloff>:/images/icons/tree-new.gif</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="bPatternView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Open CGNS/SIDS sub-trees database</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../R/Res.qrc">
      <normaloff>:/images/icons/pattern.gif</normaloff>:/images/icons/pattern.gif</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="bOptionView">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set user defined options</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../R/Res.qrc">
      <normaloff>:/images/icons/options-view.gif</normaloff>:/images/icons/options-view.gif</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="bAbout">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>About....</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../R/Res.qrc">
      <normaloff>:/images/icons/view-help.gif</normaloff>:/images/icons/view-help.gif</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bClose">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../R/Res.qrc">
      <normaloff>:/images/icons/close-view.gif</normaloff>:/images/icons/close-view.gif</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../R/Res.qrc"/>
 </resources>
 <connections/>
</ui>
